internal/store/utils.go

```diff
@@ -58,30 +63,18 @@ func addConditionMetrics(cs v1.ConditionStatus) []*metric.Metric {
 	return ms
 }
 
-func kubeLabelsToPrometheusLabels(labels map[string]string) ([]string, []string) {
-	labelKeys := make([]string, len(labels))
-	labelValues := make([]string, len(labels))
+func kubeMetadataToPrometheusLabels(keyPrefix string, metadata map[string]string) ([]string, []string) {
+	labelKeys := make([]string, len(metadata))
```

Comment:
`s/labelKeys/metadataKeys`
`s/labelValues/metadataValues`

Reply:
`sanitizeLabelName` may also refactored to `sanitizeMetadataName` to keep in sync.

Reply:
Here it has to be labelKeys and labelVaules IMO as the method returns prometheus label k-v pairs.